eng/testing/linker/trimmingTests.targets

```diff
@@ -55,26 +55,39 @@
 
     <MakeDir Directories="$(_projectDir)" />
     <WriteLinesToFile File="$(_projectFile)"
-                      Lines="$([System.IO.File]::ReadAllText('$(ProjectTemplate)').Replace('{NetCoreAppCurrent}', '$(NetCoreAppCurrent)').Replace('{RuntimePackDir}', '%(TestConsoleApps.RuntimePackDirectory)').Replace('{TargetingPackDir}','$(MicrosoftNetCoreAppRefPackDir)').Replace('{RuntimeIdentifier}','%(TestConsoleApps.TestRuntimeIdentifier)'))"
+                      Lines="$([System.IO.File]::ReadAllText('$(ProjectTemplate)').Replace('{NetCoreAppCurrent}', '$(NetCoreAppCurrent)').Replace('{RuntimePackDir}', '%(TestConsoleApps.RuntimePackDirectory)').Replace('{TargetingPackDir}','$(MicrosoftNetCoreAppRefPackDir)').Replace('{RuntimeIdentifier}','%(TestConsoleApps.TestRuntimeIdentifier)').Replace('{MicrosoftNETILLinkTasksVersion}', '$(MicrosoftNETILLinkTasksVersion)'))"
                       Overwrite="true" />
     <Copy SourceFiles="$(_projectSourceFile)"
           DestinationFolder="$(_projectDir)" />
     <Message Text="Generated $(_projectFile)" />
   </Target>
 
-  <Target Name="PublishTrimmedProjects"
+  <Target Name="RestoreProjects"
```

Comment:
We now need to run Restore out of proc from Publish, as we need to calculate the property for the linker package location and then perform another project evaluation for publish to use it.

Reply:
Couldn't we just do `dotnet build /t:PublishTrimmed` instead of invoking `dotnet msbuild`?

Reply:
mm good call that might work, let me check.

Reply:
yup that worked, pushing update...